Fits/Dreadnoughts/[FL33T] Anti-Cap Phoenix V2.md

```diff
@@ -8,43 +8,45 @@ Used to provide high DPS against other capital ships.  Highly ineffective for us
 
 ## Fit
 ```
-[Phoenix, T2 Anti-Cap Phoenix]
+[Phoenix, Anti-Cap Phoenix V2]
```

Comment:
Is the guidance enhancer worth an extra bit of range over ~10% more EHP? PDU + T2 extenders could be used in its place.

Reply:
The MGE is nice primarily because the Phoenix/PNI don't have fall-off as a failsafe if they bump with a ranged drop. The application and all that is nice as well but primarily it's for the extra range so that you're guaranteeing T2 Rage in those situations. It really amplifies the strength of the ship/weapon platform. Neither option here would be  bad, I think the extra tank certainly has uses, but honestly with shield caps the issue is cap 100% of the time if they drop on top of you because the 1 cap booster is just not enough to do any kind of meaningful tanking. A little extra buffer isn't offsetting the primary issue with the tank because your hardeners are going to be off.